src/integrals/integrals.cc

```diff
@@ -200,6 +200,22 @@ double ForteIntegrals::oei_b(size_t p, size_t q) const {
     return one_electron_integrals_b_[p * aptei_idx_ + q];
 }
 
+ambit::Tensor ForteIntegrals::oei_a_block(const std::vector<size_t>& p,
```

Comment:
Are these two function necessary? We can probably directly return a copy of py::array.

Reply:
Well, this way we get both the C++ functionality (via ambit) and the python one (via py::array).